apps/schools/domains/common/components/Icons/IconWrapper.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { CSSProperties, MouseEventHandler, SVGProps } from 'react'
+
+const SMALL_ICON_SIZE = 16
+const MEDIUM_ICON_SIZE = 20
+const LARGE_ICON_SIZE = 24
```

Comment:
must contain in consts/number

Reply:
Can we delete IconWrapper? Is it useful?